test.ts

```diff
@@ -447,6 +447,28 @@ test(function invalidBindDoesNotLeakStatements() {
   db.close();
 });
 
+test(function getColumnsFromRows() {
```

Comment:
Can you add a test for the case where the name is not specified (using an `column AS name`), so that we notice if SQLite changes it's behaviour (or at least have a chance to notice 😅)